README.md

```diff
@@ -138,33 +137,52 @@ npm run setup
 
-### Cloudant
+#### Cloudant
 
 This action will create the databases required to run the application as well as load all the design documents and indexes.  You need to have the Cloudant username and password handy, which you can find in the credentials of your Cloudant service in Bluemix.
 
-### Twitter
+#### Twitter
 
 This action will update the configuration with the parameters needed to connect to Twitter as well as subscribe to tweets.  You need to have the Consumer Key (API Key), Consumer Secret (API Secret), Access Token and Access Token Secret from dev.twitter.com available.
 
-At this point you should have decided what Twitter handle you will be "listening" to.  The Twitter handle would be something other people would tweet to, for example `@united`, `@AmericanAir`, `@JetBlue` (Please note that these are CASE SENSITIVE) etc.  This value is what Twitter would use as trigger to send you the tweets.
-
-When you selected the `Twitter` option on the main menu of the setup utility, you will be required to enter the Twitter Tokens first.  Then the screen name you are listening to, the Conversation API workspace id for the classifications and finally the Chatbot screen name and Conversation API workspace for the Dialog implementation.
+At this point you should have decided what Twitter handle you will be "listening" to.  The intents for this journey are configured to work with a Twitter account for an airline, but you could create relevant intents for whichever business domain you wish. The Twitter handle would be something other people would tweet to, for example `@aircanada`.  This value is what Twitter would use as trigger to send you the tweets.
 
-The chatbot screen name is your own Twitter account.  DO NOT USE THE SAME TWITTER SCREEN NAME YOU USED TO LISTEN TO.  This way, when you want to chat to the Chatbot via Twitter, you can use your own screen name for example `@mysuperbot blah blah blah` to send messages to your bot.
+NOTE> 
+Due to the potential for a large volume of API calls, this Accelerator will work best with a paid subscription to Bluemix.  If you try and use this accelerator on an screen name that produces a lot of tweets, then you take the risk of using up your free allocation of API calls very quickly.  The Accelerator will suspend for 15 minutes listening to tweets if there are errors returned from the enrichment pipeline.  When the receiver on the UI is paused, it usually means you have exceeded your limit for the day.
 
-If you don't want to implement a Chatbot, then leave this blank, and the Chatbot won't be started.
+When you selected the `Twitter` option on the main menu of the setup utility, you will be required to enter the Twitter Tokens first.  Then the screen name you are listening to, the Conversation API workspace id for the classifications and finally the Chatbot screen name and Conversation API workspace for the Dialog implementation.
```

Comment:
"selected" is suddenly past tense.

The last sentence here is a fragment - I'm not sure how to fix it / what the intended meaning was.

Reply:
done